packages/react-components/react-motion/stories/src/CreateMotionComponent/CreateMotionComponentImperativeRefPlayState.stories.tsx

```diff
@@ -66,6 +66,10 @@ const FadeEnter = createMotionComponent({
   keyframes: [{ opacity: 0 }, { opacity: 1 }],
   duration: motionTokens.durationSlow,
   iterations: Infinity,
+
+  reducedMotion: {
+    iterations: 1,
```

Comment:
What's the value of having a single iteration here? Is it just that this property has a clearer presence in documentation?

Reply:
With infinite iterations, the reduced motion with 1 ms was looping and flashing.

Reply:
👆 Exactly, either this or we need to complicate examples significantly.